AMW_angular/io/src/app/settings/environments/environments-page.component.html

```diff
@@ -1,48 +1,48 @@
 <div class="container w-50">
   <h1>Environments</h1>
-  <button type="button" class="btn btn-primary">
+  <button type="button" class="btn btn-primary" (click)="addDomain()">
     <app-icon icon="plus-circle"></app-icon>
     Add Domain
   </button>
-  @for(env of environmentTree(); track env.id){
+  @for(dom of environmentTree(); track dom.id){
```

Comment:
is `dom` shorthand for `domain`? 
My first association was `Document Object Model` - personally, I'd prefer `domain` instead of `dom`